assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/codenvy/che.properties

```diff
@@ -294,6 +294,9 @@ che.openshift.liveness.probe.delay=300
 che.openshift.liveness.probe.timeout=1
 che.openshift.workspaces.pvc.name=claim-che-workspace
 che.openshift.workspaces.pvc.quantity=10Gi
+che.openshift.workspace.cpu.limit=1
+# Override memory limit used for openshift workspaces. String, e.g. 1300Mi
+che.openshift.workspace.memory.limit=
```

Comment:
We could do something like this, I didn't do it initially since OpenShift interprets the string provided. So, for example, `1Gi == 1024Mi == 1073741824` are all valid values for memory. CPU is measured as e.g. 1 = 1 CPU, or 500m = 500 millicores. I wasn't sure the best way to include dimensions, since the strings are passed directly to OpenShift, so anything that is valid as a resource limit in OpenShift is valid here.